zip-codes/scripts/map.js

```diff
@@ -7,6 +7,17 @@ function initMap() {
     scrollwheel: true,
     zoom: 8
   });
-
+  if (zips.length) {
+    // zips.val() 
+    console.log(zips.val());
+  }
+  var markerCity = {lat:$(this) ,lng:$(this)};
```

Comment:
Instead of trying to access values from this file (like "this" or zips)  to create your marker, you could pass in these values when you call initMap. You should use a parameter in the function definition and reference that parameter in your Marker object (the value of the position property).